examples/README.md

```diff
@@ -380,25 +380,25 @@ Intel® Neural Compressor validated examples with multiple compression technique
     <td>ResNet18</td>
     <td>Image Recognition</td>
     <td>Post-Training Static Quantization</td>
-    <td><a href="./pytorch/image_recognition/torchvision_models/quantization/ptq/cpu/eager">eager</a> / <a href="./pytorch/image_recognition/torchvision_models/quantization/ptq/cpu/fx">fx</a> / <a href="./pytorch/image_recognition/torchvision_models/quantization/ptq/cpu/ipex">ipex</a></td>
+    <td><a href="https://github.com/intel/neural-compressor/tree/v1.14.2/examples/pytorch/image_recognition/torchvision_models/quantization/ptq/cpu/eager">eager</a> / <a href="./pytorch/image_recognition/torchvision_models/quantization/ptq/cpu/fx">fx</a> / <a href="./pytorch/image_recognition/torchvision_models/quantization/ptq/cpu/ipex">ipex</a></td>
```

Comment:
https://github.com/intel/neural-compressor/tree/**old_api_examples**

Reply:
Please update with this branch, we have some incompatible change so have to maintain a old api branch instead of use the tag.

Reply:
okay

Reply:
@chensuyue  @xin3he  I am confused... the changes I saw in this PR is using new API. why this PR needs to be old_api_examples branch?